[2022-01-31 17:28:20,848] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_time_zone.download_dataset_task scheduled__2022-01-30T00:00:00+00:00 [queued]>
[2022-01-31 17:28:20,937] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_time_zone.download_dataset_task scheduled__2022-01-30T00:00:00+00:00 [queued]>
[2022-01-31 17:28:20,940] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 17:28:20,942] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-31 17:28:20,944] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 17:28:21,024] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-01-30 00:00:00+00:00
[2022-01-31 17:28:21,039] {standard_task_runner.py:52} INFO - Started process 383 to run task
[2022-01-31 17:28:21,064] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_time_zone', 'download_dataset_task', 'scheduled__2022-01-30T00:00:00+00:00', '--job-id', '277', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_time_zone.py', '--cfg-path', '/tmp/tmpehzuwhw7', '--error-file', '/tmp/tmp2dnh02_w']
[2022-01-31 17:28:21,069] {standard_task_runner.py:77} INFO - Job 277: Subtask download_dataset_task
[2022-01-31 17:28:21,419] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_time_zone.download_dataset_task scheduled__2022-01-30T00:00:00+00:00 [running]> on host 943e1344cac5
[2022-01-31 17:28:21,702] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_time_zone
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-30T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-30T00:00:00+00:00
[2022-01-31 17:28:21,713] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-31 17:28:21,717] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/misc/taxi+_zone_lookup.csv > /opt/***/taxi+_zone_lookup.csv']
[2022-01-31 17:28:21,756] {subprocess.py:85} INFO - Output:
[2022-01-31 17:28:22,888] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-31 17:28:23,099] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_time_zone, task_id=download_dataset_task, execution_date=20220130T000000, start_date=20220131T172820, end_date=20220131T172823
[2022-01-31 17:28:23,273] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-31 17:28:23,758] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
